lib/composers/suspendComposer.js

```diff
@@ -0,0 +1,12 @@
+const {makeModifierComposer} = require('../utils/makeComposer');
+const composers = require('../constants/composer');
+
+/**
+ * Defined suspend method
+ */
+const suspendComposer = makeModifierComposer(composers.SUSPEND_APP, ['suspend'], (_, meta) => ({
+	...meta,
+	isSuspended: true,
```

Comment:
Same as for `isClosed`.

Maybe `isSuspendApp` or even just `suspendApp` / `closeApp` would be more clear for understanding.